sources/packages/backend/apps/api/src/types/form.ts

```diff
@@ -1,14 +1,5 @@
-export interface DryRunSubmissionResult {
+export interface DryRunSubmissionResult<T = any> {
```

Comment:
In the future, in a separate effort, we can change the `any` to `unknown`. Right now this is which will bring some improvement and cause less impact.
Thanks for doing this change also.